### ISSUE TITLE
Allow disabling checkout

### ISSUE THREAD

**Issue by zanieb** [acj/freebsd-firecracker-action#1]
It looks like this action unconditionally checks out the parent repository at https://github.com/acj/freebsd-firecracker-action/blob/d4386fd9a54f387b315d34759479c7cd78c56505/action.yml#L82-L84

In our use case, we don't actually need the repository. We're copying in what we need. Our complete workflow is available at https://github.com/astral-sh/uv/blob/b70c4f30eeb47c15fbabd54173f952877d5cfc43/.github/workflows/ci.yml#L604-L628

It'd be great if we could just disable the checkout. Is it needed for some reason?

See https://github.com/acj/freebsd-firecracker/issues/1 for the motivation and more context.

Thanks again!

**Comment by acj** [comment]
> Is it needed for some reason?

No, I think it was for convenience. Adding a toggle is a good idea. It might be a bit before I have time to implement this, but I'd be happy to review a PR.

**Comment by zanieb** [comment]
Thanks for the reply! I'll put a pull request up if I have the time :) the FreeBSD test is more of a convenience than a necessity for us.
